scripts/foundationdb/foundationdb.service

```diff
@@ -0,0 +1,33 @@
+# /etc/systemd/system/foundationdb.service
```

Comment:
Not really, it just seems to be good practice to use more modern systemd.
And I'm trying practicing work with it where applicable.

Reply:
Unfortunately, changing the script that systemd uses makes upgrading the FDB package more difficult in the future. If there's a crucial difference that makes the default systemd behavior unusable for our case, I can be convinced. However, I would also advocate for pushing that change upstream so that we don't need customizations in the future.